extra_data/tests/test_streamer.py

```diff
@@ -1,147 +0,0 @@
-"""Test streaming data with ZMQ interface."""
```

Comment:
Can you rebase on master so we can check that this deletion doesn't make it into the merge? There's still a much reduced `test_streamer.py` after  #50.